eggdrop.conf

```diff
@@ -1158,17 +1158,21 @@ server add ssl.example.net +7000
 #set sasl-timeout 15
 
 # To request the account-notify feature via CAP, set this to 1
-#set account-notify 0
+set account-notify 1
 
 # To request the extended-join feature via CAP, set this to 1
-#set extended-join 0
+set extended-join 1
 
 # To request the invite-notify feature via CAP, set this to 1
 #set invite-notify 0
```

Comment:
I was going to say I was wary of enabling this by default, but I talked myself out of it, this is a good idea. But that inner-monologue left me thinking we need to add a big warning above this section saying "just because you enable it here doesn't mean the server supports it" or some variation. I definitely people thinking these are enabled simply because it is set here.

Reply:
I think your warning on top of the CAP block in the configuration should be enough